WorkloadManagementSystem/Client/CPUNormalization.py

```diff
@@ -139,7 +139,8 @@ def getCPUTime( CPUNormalizationFactor ):
     if not CPUNormalizationFactor:  # if CPUNormalizationFactor passed in is 0, try get it from the local cfg
       CPUNormalizationFactor = gConfig.getValue( '/LocalSite/CPUNormalizationFactor', 0.0 )
       # if CPUNormalizationFactor is not even in the local cfg, it's a problem, and yes the next line will raise an exception
-    CPUTime = CPUTime / int( CPUNormalizationFactor )
+    if int( CPUNormalizationFactor ):
```

Comment:
I think Federico meant that it throws an exception if it is 0.0, he explained it in his pull request I believe. Did you discuss this change with him ?